azurerm/internal/services/eventhub/eventhub_resource.go

```diff
@@ -229,6 +251,10 @@ func resourceEventHubRead(d *pluginsdk.ResourceData, meta interface{}) error {
 			d.Set("message_retention", props.MessageRetentionInDays)
 			d.Set("partition_ids", props.PartitionIds)
 
+			if eventhubStatus := props.Status; eventhubStatus != nil {
+				d.Set("status", eventhubStatus)
```

Comment:
I think we should take care of this lint error and then we'll be good to go

Reply:
Updated